codalab/worker/worker_run_state.py

```diff
@@ -572,6 +643,12 @@ def _transition_from_UPLOADING_RESULTS(self, run_state):
             Move to FINALIZING state
         """
         if run_state.is_restaged:
+            log_bundle_transition(
+                run_state.bundle.uuid,
+                run_state.stage,
+                RunStage.RESTAGED,
+                'the bundle has not started uploading yet',
```

Comment:
I'm not sure what this means.

Reply:
it should be this case - https://github.com/codalab/codalab-worksheets/blob/master/codalab/worker/worker_run_state.py#L565

Reply:
Can you incorporate that into the log message? It's still unclear to me.

Reply:
I think including the link to the log message is a bit awkward. Why don't we just ask the developer to check the comments in the code?

Reply:
I just don't understand `the bundle has not started uploading yet`. Can we include a better, clearer reason?

Reply:
This explanation is incorrect. It's just checking if `run_state.is_restaged` is specified; if yes, then restage it. It comes from [here]([https://github.com/codalab/codalab-worksheets/pull/2186#discussion_r423395813]).